test/Microsoft.ML.StaticPipelineTesting/StaticPipeTests.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.ML.Transforms.Text;
 using Microsoft.ML.Data.StaticPipe;
```

Comment:
Ahhhh. So nice to see a nice fresh PR. And you know what I'm going to say as my very first comment?

You know as well as I do. I always do it of course. I'd even go so far as to call it a tradition. A hoary tradition, by this point.

Just pausing for a moment, to relish the moment. You understand. :D Because, you know, I figure I deserve it at this point. It may seem a little self indulgent, and I suppose it is, but you know, I have to do it. Because, without it, I just wouldn't be me, you feel me?

So here it is:

Do you want to sort your `using`s? :)